### PR TITLE
fix: SB-30832 and SB-30816

### PR DESCRIPTION
**Pull request by manojSRawat**

### CODE REVIEW

**Review by vinukumar-vs — DISMISSED** [review]

File: src/app/resourcebundles/data/consumption/hi.properties [hunk 1]
```diff
@@ -1990,4 +1990,6 @@ frmelmnts.lbl.of = का
 frmelmnts.lbl.boardsTest=Board {frameworkCategory1}
 frmelmnts.lbl.mediumTest=Medium {frameworkCategory2}
 frmelmnts.lbl.gradeTest=Grade {frameworkCategory3}
-frmelmnts.lbl.boardsTestWithText=बोर्ड {frameworkCategory3}
\ No newline at end of file
+frmelmnts.lbl.boardsTestWithText=बोर्ड {frameworkCategory3}
```
Comment (rajeevsathish) [inline src/app/resourcebundles/data/consumption/hi.properties:1993]: with out the JSON file the properties file will not work

**Review by rajeevsathish — CHANGES_REQUESTED** [review]

File: src/app/resourcebundles/data/consumption/hi.properties (hunk 1 above)
Comment (manojSRawat) [reply src/app/resourcebundles/data/consumption/hi.properties:1993]: we only added the last line `frmelmnts.lbl.questionSets = प्रश्न सेट`. Above mentioned line is already there

Comment (tarang27) [reply src/app/resourcebundles/data/consumption/hi.properties:1993]: @rajeevsathish can you please merge this PR. As Manoj mentioned we have not added the line on which you have shared the comment. 
Thanks

**Review by rajeevsathish — CHANGES_REQUESTED** [review]
If only hi.properties has change then please push only hi.json not all

**Pull request merged**